CONTRIBUTING.md

```diff
@@ -0,0 +1,123 @@
+# Contributing guide 
+
+Contributions and feedback on your experience of using this software are welcome.
+
+This includes bug reports, feature requests, ideas, pull requests, and examples of how you have used this software.
+
+Please see the [Code of Conduct](CODE_OF_CONDUCT.md) and follow any templates configured in GitHub when reporting bugs, requesting enhancements, or contributing code.
+
+Please raise any significant new functionality or breaking change an issue for discussion before raising a Pull Request for it.
+
+## For contributors
+
+Anyone can be a contributor. Either you found a typo, or you have an awesome feature request you could implement, we encourage you to create a Pull Request.
+
+### Pull Requests
+
+- The latest changes are always in `canary`, so please make your Pull Request against that branch.
+- Pull Requests should be raised for any change
+- Pull Requests need approval of a [core contributor](https://next-auth.js.org/contributors#core-team) before merging
```

Comment:
PRs in this repo can be accepted by the adapters' respective maintainers. We could add a CODEOWNERS file to the repo to automatically assign maintainers to certain adapter PRs https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners